### PR TITLE
Portfolio admin reskin

### PR DESCRIPTION
**Pull request by leigh-mil**
## Description
Updates to the Portfolio Admin page based on the new designs. 
As per a conversation with Upma, I changed the url from `/edit` to `/admin` and removed the `Cancel` button. 

## Pivotal
[https://www.pivotaltracker.com/story/show/163834239](https://www.pivotaltracker.com/story/show/163834239)

## Screenshots
screen shot 2019-02-11 at 4 38 31 pm

### CODE REVIEW

File: atst/routes/portfolios/index.py [hunk 1]
```diff
@@ -18,11 +18,22 @@ def portfolios():
...
-    return render_template("portfolios/edit.html", form=form, portfolio=portfolio)
+    pagination_opts = Paginator.get_pagination_opts(http_request)
+    audit_events = AuditLog.get_portfolio_events(
+        g.current_user, portfolio, pagination_opts
+    )
+    return render_template(
+        "portfolios/edit.html",
+        form=form,
+        portfolio=portfolio,
+        portfolio_name=portfolio.name,
```
Comment (patricksmithdds) [inline atst/routes/portfolios/index.py]: I don't think these `portfolio_*` args are necessary if you're passing the `portfolio` into `render_template`.

File: templates/fragments/audit_events_log.html [hunk 2]
```diff
@@ -1,10 +1,7 @@
 {% from "components/pagination.html" import Pagination %}
 
-<section class="block-list">
-  <header class="block-list__header">
-    <h1 class="block-list__title">{{ "audit_log.header_title" | translate }}</h1>
-  </header>
-
+<section class="block-list activity-log">
+  <div class='subheading'>Activity Log</div>
```
Comment (drummond-dds) [inline templates/fragments/audit_events_log.html]: Should this be a translation?

File: styles/elements/_block_lists.scss [hunk 3]
```diff
@@ -15,6 +16,11 @@
   display: flex;
   flex-direction: row;
   justify-content: space-between;
+  background-color: $color-gray-lightest;
+  padding: $gap 2 * $gap;
+  border-top: none;
+  border-bottom: none;
```
Comment (patricksmithdds) [inline styles/elements/_block_lists.scss]: I think this affected the applications list, as well -- could you update the styles on the applications to add more padding?

Before:

image

After:

image

The toggle buttons are too close to the top border now.

File: styles/elements/_block_lists.scss [hunk 4]
```diff
@@ -1,5 +1,6 @@
 @mixin block-list {
   @include panel-margin;
+  box-shadow: 0 6px 18px 0 rgba(144,164,183,0.3);
```
Comment (drummond-dds) [inline styles/elements/_block_lists.scss]: We have this `rgba(144,164,183,0.3)` in a few places now.  I wonder if we should extract it out to a variable like we have for `$color-white` etc.

File: styles/elements/_block_lists.scss (hunk 3 above)
Comment (leigh-mil) [reply styles/elements/_block_lists.scss]: @patricksmithdds Good catch!

File: styles/elements/_block_lists.scss (hunk 4 above)
Comment (leigh-mil) [reply styles/elements/_block_lists.scss]: I commented the same thing on Patrick's PR!

**Pull request merged**